nservicebus/upgrades/7to8/index.md

```diff
@@ -11,6 +11,21 @@ upgradeGuideCoreVersions:
 
 NOTE: This is a working document; there is currently no timeline for the release of NServiceBus version 8.0.
 
+## Support for external dependency injection containers
+
+Support for external dependency injection containers is no longer provided by NServiceBus adapters for each container library. Instead, the [`NServiceBus.Extensions.DependencyInjection` package](/nservicebus/dependency-injection/extensions-dependencyinjection.md) provides the ability to use any container that conforms to the `Microsoft.Extensions.DependencyInjection` container abstraction.
+
+The following adapter packages will no longer be provided:
+
+* [Autofac](/nservicebus/dependency-injection/autofac.md)
+* [Castle](/nservicebus/dependency-injection/castlewindsor.md)
+* [StructureMap](/nservicebus/dependency-injection/structuremap.md)
+* [Unity](/nservicebus/dependency-injection/unity.md)
+
+### Property injection
+
+NServiceBus Version 7 enabled [automatic property injection](/nservicebus/dependency-injection/extensions-dependencyinjection.md#property-injection), this is not possible via `Microsoft.Extensions.DependencyInjection` and must be explicitly enabled using the native API of containers that supports it.
```

Comment:
late to the party, but that hasn't been an NSB 7 related feature or was it?

Reply:
I meant "enables", will update